packages/app/src/__tests__/NexusPack.test.tsx

```diff
@@ -220,8 +225,9 @@ test('visualize NXdata group with unknown interpretation', async () => {
   await selectExplorerNode('nexus_malformed/interpretation_unknown');
 
   const tabs = await findVisSelectorTabs();
-  expect(tabs).toHaveLength(1);
-  expect(tabs[0]).toHaveTextContent(NexusVis.NxImage); // support check falls back to signal dataset dimensions
+  expect(tabs).toHaveLength(2); // support check falls back to signal dataset dimensions (4D)
```

Comment:
This comment and the one below would gain to be a bit more explicit. For example, simply add `(4D supports both Image and Spectrum)`